gulpfile.js

```diff
@@ -86,6 +86,10 @@ gulp.task('types', function(done) {
 
   // write contents to 'built/index.d.ts'
   fs.writeFileSync(outputFile, contents);
+
+  // copy over config.ts file so users just need to import from 'protractor/config';
```

Comment:
1. My comment should say `import {Config} from 'protractor/configuration';` instead...but we could also do `import {Config} from 'protractor/config';`
2. `/lib/globals.d.ts` is only for protractor internals to set up global variables / fix some of the missing interfaces. `/globals.ts` exports variables from variables in the global namespace. So users can do `import {browser} from 'protractor/globals';`